tests/unit/metrics/ZenkoMetrics.spec.js

```diff
@@ -94,8 +94,9 @@ describe('ZenkoMetrics', () => {
             'pet_counter{type="puppy"} 2',
         ];
         const lines = {};
-        ZenkoMetrics.asPrometheus().split('\n').forEach(line => {
-            lines[line.trimRight()] = true;
+        const metrics = await ZenkoMetrics.asPrometheus();
```

Comment:
`ZenkoMetrics.asPrometheus` isn't declared `async`, or is it another one?

Reply:
Nope, I missed it.